src/components/Remove.js

```diff
@@ -1,7 +1,18 @@
 import React from 'react';
+/* import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/book'; */
 
 export default function Remove() {
+  /* const dispatch = useDispatch();
+
+  const remBook = (e) => {
+    const { id } = e.target;
+    dispatch(removeBook({ id }));
+  }; */
+
   return (
-    <div>Remove</div>
+    <div>
+      {/* <button type="button" id={book.id} onClick={remBook}>Remove</button> */}
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.